x/subscription/types/subscription.go

```diff
@@ -7,6 +7,10 @@ import (
 	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
 )
 
+const (
+	MAX_SUBSCRIPTION_DURATION = 12 // max duration of subscription in months
```

Comment:
why 12 is the max? if i want 13 months i can't?

Reply:
That's the design decision.
Rationale: this is similar to standard SaaS offering, you can pay per month or per year, but not for years onward in advance (so users won't try to hedge on costs going up in coming years, and providers won't be caught in unreasonable commitment to provide services for very long time at fixed, low cost).